src/index.ts

```diff
@@ -418,12 +316,10 @@ export default class Mnoga {
     const newTranslations = { ...translations };
 
     for(const context in newTranslations) {
-      if (process.env.NODE_ENV !== 'production') {
```

Comment:
I got rid of these node environment variables and just have it throw errors now. Personally, I think it's easier to enforce the rules since the utilities are very extensible.